src/gtk/widgets/recent_info.rs

```diff
@@ -67,15 +67,15 @@ impl RecentInfo {
     }
 
     pub fn get_added(&self) -> i64 {
-        unsafe { ffi::gtk_recent_info_get_added(GTK_RECENT_INFO(self.unwrap_widget())) }
+        unsafe { ffi::gtk_recent_info_get_added(GTK_RECENT_INFO(self.unwrap_widget())) as i64 }
```

Comment:
The following patch will fix this:

 - [0004-GtkRecentInfo-use-Option-u64-instead-of-i64.patch](http://www.synkhronix.com/pub/rust/patches/rgtk/0004-GtkRecentInfo-use-Option-u64-instead-of-i64.patch)